src/ubx.cpp

```diff
@@ -428,7 +428,7 @@ int GPSDriverUBX::configureDevicePreV27()
 		return -1;
 	}
 
-	if (!configureMessageRateAndAck(UBX_MSG_NAV_SVINFO, (_satellite_info != nullptr) ? 5 : 0, true)) {
+	if (!configureMessageRateAndAck(UBX_MSG_NAV_SVINFO, 5, true)) {
```

Comment:
Thanks!  I changed that while bench testing but left it to standardize it with the lines above. I didn't think the impact over the UART would mean anything as it is the same as current behavior when the message is populated and no data otherwise; happy to revert it though.

Reply:
Yes please revert.

Reply:
Done yesterday. Thanks!